docker-compose.yaml

```diff
@@ -27,9 +27,9 @@ services:
       - POSTGRES_USER=$POSTGRES_DB_USER
       - POSTGRES_PASSWORD=$POSTGRES_DB_PASSWORD
       - REPLICATOR_SYNC_PERIOD=$REPLICATOR_SYNC_PERIOD
+    command:
+      - sh -c 'wait-for postgres:5432 -- echo "The replication can start"'
     depends_on:
       - postgres
-    links:
```

Comment:
Maybe we should add a shared network?

Reply:
Isn't that done out-of-box by `docker-compose`? When you start docker-compose, there's a default network created.

Reply:
Yeh, I think that is correct. So this comment can be ignored I think.